lib/dialects/mssql/index.js

```diff
@@ -31,7 +31,6 @@ function Client_MSSQL(config = {}) {
     min: 1,
     max: 1,
     idleTimeoutMillis: Number.MAX_SAFE_INTEGER,
-    evictionRunIntervalMillis: 0,
```

Comment:
Why remove it instead of replacing with reapIntervalMillis?

Reply:
Considering this hasn't been doing anything for the current version of mssql, I assumed it wasn't needed.

I can add `reapIntervalMillis` if you like?

Reply:
@dhensby I'm not sure why it was disabled in the first place, could be related to mssql always still having its own pool under-the-hood, which I'm not sure is still the case. Wonder if we double-pool it still when used from knex. If not, this whole default config could be dropped.

Reply:
@dhensby Probably the best solution here would be to disable knex pooling altogether when using node-mssql, and just use your pool, but I'm not sure if you expose it in a way that would allow us to drop-in it where we have ours now.

Reply:
We're using tarn under the hood and it's exposed via `.pool` property on our pool, which would give your library direct access to the tarn pool if that's what you needed? Though it's tarn v1 in mssql v6 and tarn v3 in mssql v7.

However, that's not something I have the time to do right now and was hoping I could just provide a quick fix to prevent errors in mssql v7 while continuing to provide support for v6.

Reply:
@dhensby Fair point, and this sounds like the kind of an optimization we should so ourselves on our end. Within the scope of this PR let's just remove this parameter and drop support for v4.